clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -84,7 +84,7 @@ const checkDefaultExperienceRender = () => {
     cy.get("input").should("be.checked");
   });
 
-  cy.get("#fides-tab-Features").click();
+  cy.get("#fides-tab-features").click();
```

Comment:
This was an intentional refactor to avoid CSS IDs/classes that included _localized_ strings